runner/analyze.py

```diff
@@ -128,6 +79,15 @@ def _perform_initialization_analysis(
     train_set: Imageset,
     cnn_analysis: Dict[str, Dict[str, FloatArray]],
 ):
+    summary = brain.scan()
+    filepath = os.path.join(cfg.system.run_dir, "brain_summary.txt")
+
+    with open(filepath, "w") as f:
+        f.write(summary)
+
+    if cfg.simulation.use_wandb:
+        wandb.save(filepath, base_path=cfg.system.run_dir, policy="now")
```

Comment:
filepath already contains the base_path, if I'm not misunderstanding sth

Reply:
`base_path` tells wandb what it should _drop_ from `filepath`